### PR TITLE
Added Capitalize in Javascript

### LINKED ISSUE #1067: Add Capitalize in JavaScript Article

### PR DESCRIPTION
**Pull request by Daniellunsc**
- [X] I fixed #1067

### Code Snippets

- [X] I named the pull request using `Added/Updated <Sample Program> in <Language>` format
- [X] I created/updated the language README
  - [X] I added the sample program name to the README
  - [X] I added reference link(s) to the README

## Notes

I'm a little confuse about the workflow to contribute.

### CODE REVIEW

**Comment by Daniellunsc** [comment]
Of course! I would enjoy creating this document. 
Just to clarify, every program should have one document like this?

**Comment by jrg94** [comment]
Yup! That way if someone wants to come along and implement Capitalize in Python or some other language we have an agreed upon standard for testing.

**Comment by Daniellunsc** [comment]
Nice, i'll be back with the document 😄

**Comment by jrg94** [comment]
[Here's a template to help][1]! 🛩 

[1]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/PROJECT_ARTICLE_TEMPLATE.md

File: docs/projects/capitalize/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Requirements
+
+For this sample program, each solution should return the string with the first letter in uppercase.
+
+```
+string -> String
+react -> React
+java -> Java
+car -> Car
+a long string term: A long string term
```
Comment (jrg94) [inline docs/projects/capitalize/index.md]: For consistency:

```suggestion
a long string term -> A long string term
```

File: docs/projects/capitalize/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: Capitalize in Every Language
+layout: default
+date: 2019-03-31
+last-modified: 2019-03-31
+featured-image:
+tags: [capitalize]
+authors:
```
Comment (jrg94) [inline docs/projects/capitalize/index.md]: You can add yourself here! We have an authors doc: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml. Put whatever you want there and then place your username here like:

```yml
authors:
  - your_username
```

**Comment by Daniellunsc** [comment]
> Looks great. The only suggestion I have is to add a few examples in a table in the testing section, but I think we can add those when we get around to testing.

Thanks! I was confused about the workflow but you clarified to me. I added the author, but i'm not sure if its right. Awaiting your review.

**Pull request merged**